v2/dtos/reading.go

```diff
@@ -5,7 +5,9 @@
 
 package dtos
 
-import "github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+)
```

Comment:
In Core WG we decided that all DTOs need to be versioned. So please add version to Reading. Make sense that Request DTOs don't need to be versioned.

Reply:
Version added.